src/current/advisories/a99561.md

```diff
@@ -12,13 +12,13 @@ Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
 
 ## Description
 
-A primary [index](../v23.1/indexes.html) is automatically created for each table, which indexes the table's [primary key](../v23.1/primary-key.html), or another unique value when there is no primary key. In CockroachDB v20.2 and v21.1 (the first two versions to support [`ALTER PRIMARY KEY`](../v21.1/alter-primary-key.html)), there was a bug where in some cases the index, which is automatically created to provide uniqueness for the existing primary key, would have its `EncodingType` field marked as `PrimaryIndexEncoding`. This behavior was fixed in [#66599](https://github.com/cockroachdb/cockroach/pull/66599), but left [secondary indexes](../v23.1/schema-design-indexes.html) with primary encoding.
+A primary [index](../v23.1/indexes.html) is automatically created for each table, which indexes the table's [primary key](../v23.1/primary-key.html), or another unique value when there is no primary key. In CockroachDB v20.2 and v21.1 (the first two versions to support [`ALTER PRIMARY KEY`](../v21.1/alter-primary-key.html)), there was a bug where in some cases the automatically created index would have its `EncodingType` field marked as `PrimaryIndexEncoding`. This behavior was fixed in [#66599](https://github.com/cockroachdb/cockroach/pull/66599), but left [secondary indexes](../v23.1/schema-design-indexes.html) with primary encoding.
```

Comment:
I'm still confused. We start by talking about primary indexes, and we finish with talking about secondary indexes that look like primary indexes. I think we need to explain more whether the bug is with primary indexes or with secondary indexes.

Reply:
@mdlinville I _think_ I see the problem....

Automatically created indexes can refer to the secondary index as well. That is, columns where there is a `UNIQUE` constraint, and therefore CRDB auto creates a secondary index. I think the first sentence/second sentence should also include this information re secondary indexes. 

Perhaps @rimadeodhar when you are making further amendments you could confirm this?